CLR/Debugger/Debugger.cpp

```diff
@@ -622,7 +622,7 @@ bool CLR_DBG_Debugger::AccessMemory( CLR_UINT32 location, UINT32 lengthInBytes,
                 {
                     case AccessMemory_Check:
                     case AccessMemory_Read:
-                        if(deviceInfo->Attribute.SupportsXIP)
+                        if (deviceInfo->Attribute.SupportsXIP)

```

Comment:
Please don't add whitespace there like that
The original was fine as it was, and the preferred style